frontend/src/components/donation_page/index.jsx

```diff
@@ -32,6 +32,33 @@ class Donate extends Component {
             Wright, Founder/Executive Director at (773) 941-1200 or via email at{" "}
             <strong>Founder@TheYMIM.org</strong>
           </p>
+
+          <div className="mx-auto">
+            <form action="https://www.paypal.com/" method="post" target="_top">
+              <input type="hidden" name="cmd" value="_s-xclick" />
```

Comment:
Please replace this line with this instead so we can send it to the YMIM PayPal Page

```
<input type="hidden" name="cmd" value="_s-xclick" />
<input type="hidden" name="hosted_button_id" value="34L62HLMP5VEC" />
```